modules/oadp-release-notes-1-3-2.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes-1-3.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oadp-release-notes-1-3-2_{context}"]
+= {oadp-short} 1.3.2 release notes
+
+The {oadp-first} 1.3.2 release notes list resolved issues and known issues.
+
+//[id="new-features-1-3-2_{context}"]
+//== New features
+
+
+[id="resolved-issues-1-3-2_{context}"]
+== Resolved issues
+
+.DPA fails to reconcile if a valid custom secret is used for BSL
+
+DPA fails to reconcile if a valid custom secret is used for backup storage location (BSL), but the default secret is missing. The workaround is to create the required default `cloud-credentials` initially. When the custom secret is re-created, it can be used and checked for its existence. link:https://issues.redhat.com/browse/OADP-3193[(OADP-3193)]
```

Comment:
```suggestion
DPA fails to reconcile if a valid custom secret is used for Backup Storage Location (BSL), but the default secret is missing. The workaround is to create the required default `cloud-credentials` initially. When the custom secret is re-created, it can be used and checked for its existence. 

link:https://issues.redhat.com/browse/OADP-3193[OADP-3193]
```